examples/resources/hpegl_vmaas_transport_zone/resource.tf

```diff
@@ -0,0 +1,6 @@
+# (C) Copyright 2021 Hewlett Packard Enterprise Development LP
+
+data "hpegl_vmaas_transport_zone" "tf_zone" {
+  name         = "nsx_transport_zone_overlay"
+  service_name = "NSX-T"
```

Comment:
do we really need this field here? are we using it ?

Reply:
yeah, it is using in network